innkeeper/eskiptojson.go

```diff
@@ -0,0 +1,125 @@
+package innkeeper
+
+import (
+	"github.com/zalando/skipper/eskip"
+	"github.com/zalando/skipper/routeid"
+)
+
+func convertPathMatcher(r *eskip.Route) *pathMatcher {
+	var (
+		pathMatch     string
+		pathMatchType matchType
+	)
+
+	if r.Path != "" {
+		pathMatch = r.Path
+		pathMatchType = matchStrict
+	} else if len(r.PathRegexps) > 0 {
+		// TODO we should only have one path regexp
```

Comment:
agree, this could be changed generally. Preferably in a separate pull request. We can shoot a `log.Warning` until then, in the cases of multiple PathRegexps.